src/Blazor.Gitter.Core/Components/Shared/MainLayout.razor

```diff
@@ -1,6 +1,6 @@
 ﻿@inherits MainLayoutModel
 
-<BlazorEmbedLibrary.EmbeddedContent Assemblies="@AssemblyList" />
+<BlazorEmbedLibrary.EmbeddedContent Assemblies="@AssemblyList" Debug="true"/>

```

Comment:
Can you please make sure Debug is removed before we merge.

Reply:
Absolutely